diagnostic.md

```diff
@@ -7,40 +7,43 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone https://github.com/conorjennings/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
+Change into the new directory from the clone command
+ cd git-diagnostic/
 
-<!-- Remove this comment and place your answer here. -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
-```
+I used the following to ensure i go into the correct branch:
+git checkout -b response
+
+The other approach is 2 steps as follows:
+git branch response
+git checkout response
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
-```
+git status
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
-```sh
-<!-- Remove this comment and place your answer here. -->
-```
+git add <fileName>
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+git commit <nameOfFile>
+This will then open Atom and I can put in the relevant checkin comments. Then hit save and exit out to return the command prompt in terminal. From a formating, comment, the assignment(project) number and what has changed in this commit.
```

Comment:
Close here Conor, after you have added your files to the staging area you don't need to specify a file name in the commit. You can just run `git commit` to commit all the files that have been changed.

The two formatting items that you need in the commit are the Title of the commit, a short summary of what you did, and then a commit body that contains a message with more details about the changes you have made.